lib/rouge/lexers/alda.rb

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class Alda < RegexLexer
+      title "Alda"
+      desc "The Alda programming language for music composition"
+      tag 'alda'
+      filenames "*.alda"
+      mimetypes 'text/alda'
+
+      def self.detect?(text)
+        return true if text.shebang? 'alda'
+      end
+
+      state :root do
+        rule /#[ \S]+/, Comment
```

Comment:
I only had a very quick look at the docs so I may have missed it but does the hash mark really need to be followed by a space and not any other form of whitespace?